### DISCUSSION TITLE
Let `ManiaModCover` ranked

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#30831] [Ideas] [answered]
The functions of this mod can be completely realized by skins. If the play realized by skins can be ranked, then this mod should also be ranked.

**Comment by cl8n** [comment]
that's actually why I *don't* want it to be ranked... I think any static lane covers should be in the domain of skinning rather than mods

**Reply by cdwcgt** [reply]
But in fact, it doesn't need to change his skin for me. 

This is very convenient for people who can't change their skin o who want to switch or adjust the "cover" freely.

**Reply by cl8n** [reply]
adding covers in skins doesn't have great UX right now, I'm just saying that's where I'd ideally want this functionality to be / where I believe it fits best. so leaning into further support of the Cover mod is the wrong direction imo

**Comment by cdwcgt** [comment, marked as answer]
https://github.com/ppy/osu/pull/36021
wdym bro...
